lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Wallets.hs

```diff
@@ -1115,9 +1115,10 @@ spec = describe "SHELLEY_WALLETS" $ do
         let dummyChainCode = BS.replicate 32 0
         let sig = CC.xsignature $ BL.toStrict $ getFromResponse id rSig
         let key = unsafeXPub $ fst (getFromResponse #getApiVerificationKey rKey) <> dummyChainCode
-        let msg = unsafeFromHex "A10071706C65617365207369676E20746869732E"
+        let _msg = unsafeFromHex "A10071706C65617365207369676E20746869732E"
```

Comment:
this one probably we want to delete, right?

Reply:
Yeah, the string is already shown in the comment above. I deleted it.